### PR TITLE
Remove sass

### PR DESCRIPTION
**Pull request by gesa**
### Summary

node-sass has the kind of vulnerability that… may take a moment to resolve:  

Seems as good a time as any to swap out Sass for the more javascript-y less anyway.

### Checklist

- [x] Added a changelog entry

### CODE REVIEW

File: src/less/three-d-secure.less [hunk 1]
```diff
@@ -35,8 +35,6 @@
     border-radius: 3px;
     margin: auto;
     height: 450px;
-    width: 400px;
```
Comment (hollabaq86) [inline src/less/three-d-secure.less]: why are these getting removed?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 =========
 
+Unreleased
+------
+- Replace css preprocessor, from sass to less
```
Comment (hollabaq86) [inline CHANGELOG.md]: Can we add links to `sass` and `less` repo pages?

File: src/less/three-d-secure.less (hunk 1 above)
Comment (crookedneighbor) [reply src/less/three-d-secure.less]: I'm curious about that too.

File: CHANGELOG.md (hunk 2 above)
Comment (crookedneighbor) [reply CHANGELOG.md]: Ditto

File: src/less/three-d-secure.less (hunk 1 above)
Comment (gesa) [reply src/less/three-d-secure.less]: See lines 33-34 of this file (duplicates)

Comment (crookedneighbor) [reply src/less/three-d-secure.less]: :+1:

Comment (hollabaq86) [reply src/less/three-d-secure.less]: Ahhhhhhh thanks, GH cut that part off

**Comment by gesa** [comment]
The only option I considered outside of Less was vanilla CSS, but CSS variables have a long way to go, and we utilize variables quite a bit. Also nesting/un-nesting is a task fraught with potential human error, where Sass -> Less conversion is pretty easy. On that note, I diffed built css between the two preprocessors, there were no differences outside of whitespace.

**Pull request merged**